### PR TITLE
Delete the redundant define test

### PR DESCRIPTION
**Pull request by qingsenLi**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>

/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/kubelet/server/server_test.go [hunk 1]
```diff
@@ -1221,7 +1221,6 @@ func testExecAttach(t *testing.T, verb string) {
 	}
 
 	for desc, test := range tests {
-		test := test
```
Comment (draveness) [inline pkg/kubelet/server/server_test.go]: it would be better with

```go
for desc := range tests {
   test := tests[desc]
}
```

Comment (draveness) [reply pkg/kubelet/server/server_test.go]: PTAL at the failing test. The current commit introduces data race problem.

**Comment by draveness** [comment]
> /test pull-kubernetes-bazel-test

It is not flaky, and the test fails due to this commit, please take a look at the comments above.

**Comment by qingsenLi** [comment]
@draveness Thanks for review.I updated the PR based on your suggestions.

**Comment by draveness** [comment]
/lgtm

Hi @yujuhong, this is an easy one, please take a look.

**Comment by qingsenLi** [comment]
/assign @yujuhong
PTAL.thanks

**Review by thockin — COMMENTED** [review]
Thanks!

/lgtm
/approve

**Pull request merged**